include/megaapi_impl.h

```diff
@@ -1547,6 +1548,44 @@ class MegaUserAlertListPrivate : public MegaUserAlertList
     int s;
 };
 
+class MegaRecentActionBucketPrivate : public MegaRecentActionBucket
+{
+public:
+    MegaRecentActionBucketPrivate(MegaClient::recentaction& ra, MegaClient* mc);
+    MegaRecentActionBucketPrivate(int64_t timestamp, const string& user, handle parent, bool update, bool media, MegaNodeList*);
+    virtual ~MegaRecentActionBucketPrivate();
+    virtual MegaRecentActionBucket *copy() const;
+    virtual int64_t getTimestamp() const;
+    virtual const char* getUserEmail() const;
+    virtual MegaHandle getParentHandle() const;
+    virtual bool getIsUpdate() const;
+    virtual bool getIsMedia() const;
+    MegaNodeList* getNodes();
```

Comment:
It misses the virtual, and could be defined as `const`, both the method and the returned pointer :)

Reply:
excellent point :-)

Reply:
in the update I've added the override keyword, which is even better since any change in the parent class that causes the function to not override anymore will cause the compiler to alert us to the problem.  cheers!

Reply:
We should start using `overrided` now that C++11 is a must. I'll tell the team ;)

Reply:
👍